environment.staging

```diff
@@ -14,7 +14,7 @@ DCP_DOMAIN=${DSS_DEPLOYMENT_STAGE}.data.humancellatlas.org
 OIDC_AUDIENCE=https://dev.data.humancellatlas.org/,https://${DCP_DOMAIN}/
 API_DOMAIN_NAME="dss.${DCP_DOMAIN}"
 DSS_ZONE_NAME="staging.data.humancellatlas.org."
-ACM_CERTIFICATE_IDENTIFIER="f4f2b043-41ba-4116-b2f0-8d4296c3b97b"
+ACM_CERTIFICATE_IDENTIFIER="99fff90e-6ff5-44a5-852e-67c78f88c1f1"
```

Comment:
Is this addressed in  #2538 ?

Reply:
closed #2538 proper SOP is merge into master and cherry pick into staging.